### PR TITLE
created readme file

### PR DESCRIPTION
**Pull request by QQBratka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+* Apache Tomcat (v9.0.55)
+* JDBC
+* MySQL
+* Servlet
+* JSP
+* JSTL
+* HTML, CSS
+* Maven
+* Maven checkstyle plugin
+
```
Comment (PineappleApplePinecone) [inline README.md]: I think this spaces is not necessary

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+## Recommendations to run project:
+
```
Comment (PineappleApplePinecone) [inline README.md]: here

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
 # Taxi service
+
+This is a simple web app for simulating taxi service based on Stateful Web Service,
+at first you have to pass authentication or create a new driver which login and password
+will be your for authentication to the service.
+
```
Comment (PineappleApplePinecone) [inline README.md]: here and so on

**Review by PineappleApplePinecone — CHANGES_REQUESTED** [review]

**Review by PineappleApplePinecone — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
 # Taxi service
+This is a simple web app for simulating taxi service based on Stateful Web Service,
+at first you have to pass authentication or create a new driver which login and password
+will be your for authentication to the service.
+## Project description:
```
Comment (Nick97-git) [inline README.md]:
```suggestion
## Available functions:
```

File: README.md [hunk 5]
```diff
@@ -1 +1,38 @@
...
+* Ability to get all drivers, manufacturers or cars
+* Ability to add driver by driver id to car by car id
+* Ability to see your current cars
+## Implementation details in project:
+* 3 layer architecture
+    - Controllers
+    - Services
+    - DAO
+* Dependency injection and Inversion of control
+* Relations many to one, many to many
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 6]
```diff
@@ -1 +1,38 @@
...
+## Technologies used in project:
+* Apache Tomcat (v9.0.55)
+* JDBC
+* MySQL
+* Servlet
+* JSP
+* JSTL
+* HTML, CSS
+* Maven
+* Maven checkstyle plugin
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 7]
```diff
@@ -1 +1,38 @@
...
+## Recommendations to run project:
+* Intellij IDEA Ultimate Edition
+* Apache Tomcat version 9.0.55
+* MySQL and MySQL Workbench
+* Create a schema like in resources/init_db.sql in MySQL Workbench
+* Add your DB Properties in ConnectionUtil class
+* Edit your Tomcat configuration
+* After running tomcat go to http://localhost:your port and create a new driver
+* For using logger change full path in log4j2.xml file
```
Comment (Nick97-git) [inline README.md]: make a numbered list of it

File: README.md [hunk 8]
```diff
@@ -1 +1,38 @@
...
+* JSP
+* JSTL
+* HTML, CSS
+* Maven
+* Maven checkstyle plugin
+## Recommendations to run project:
+* Intellij IDEA Ultimate Edition
+* Apache Tomcat version 9.0.55
+* MySQL and MySQL Workbench
+* Create a schema like in resources/init_db.sql in MySQL Workbench
```
Comment (Nick97-git) [inline README.md]:
```suggestion
* Create a schema in MySQL Workbench using resources/init_db.sql
```

File: README.md [hunk 9]
```diff
@@ -1 +1,38 @@
...
+* Servlet
+* JSP
+* JSTL
+* HTML, CSS
+* Maven
+* Maven checkstyle plugin
+## Recommendations to run project:
+* Intellij IDEA Ultimate Edition
+* Apache Tomcat version 9.0.55
+* MySQL and MySQL Workbench
```
Comment (Nick97-git) [inline README.md]: what to do with them install or something else?

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns
- GetMyCurrentCarsController -> Long id -> Long driverId

**Comment by Nick97-git** [comment]
what about comments not related to Readme?

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**